Automattic/vip-go-mu-plugins
# Add admin notice for WP 5.6 RC1

## Description

Adds a WP admin notice to publicize the availability of WP 5.6 RC1. 5.6 will be released on December 8th and we encourage all customers to thoroughly test against the latest Release Candidate.

This message will be tweaked prior to deploying this, after we have a Lobby Post announcing the same.

The display conditions for this message are:

* Date is between now and December 9th (day after 5.6)
* WP version is less than 5.6
* User has the `manage_options` capability (which usually means administrator)

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).
- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR
1. Log in as Administrator user
1. Go to `wp-admin` - New notice should display
1. Dismiss notice and reload - should stay dismissed
1. Log in as a non-admin user (Editor)
1. Notice should _not_ display
